index.js

```diff
@@ -0,0 +1,42 @@
+let competitions = [
+    ["HTML", "C#"],
+    ["C#", "Python"],
+    ["Python", "HTML"],
+    
+];
+
+let results = [0, 0, 1];
+
+function findWinner(competitions, results) {
+    let scores = {};
+    let currentBestTeam = '';
```

Comment:
extra challenge: how to do this by using only two vars instead of 3 like you have scoreOfTeam, maxScore, winTeam, to decrease some space usage.